src/system-plugins/software-update-alert/public/webcomponents/orov-software-update-router.html

```diff
@@ -24,6 +24,8 @@
               status.progress;
 
               switch(status.progress){
+                  case "Examining":
+                  break;
```

Comment:
No issues here (tested). I don't think there is any form of meaningful whitespace in js.